proxy.conf.json

```diff
@@ -1,36 +1,36 @@
 {
```

Comment:
Is this file required only for dev env? I see with every update there is IP change and also it's static IP..Can we just ignore?

Reply:
Yes please ignore.
This file need not be updated.

Reply:
Ok

Reply:
Please remove it from PR..